### DISCUSSION TITLE
Project examples?

### DISCUSSION THREAD

**Discussion by plpullen** [keathley/wallaby#708] [General]
Hello all - I'm pretty new to coding and therefore Wallaby too, and I'm trying to find projects that have already implemented it, so I can better understand how everything works together. I figured I would ask and see if anyone knows of public projects I can look at. Cheers!

**Comment by mhanberg** [comment]
https://sourcegraph.com/search?q=context:global+%7B:wallaby%2C+%22%7E%3E+lang:elixir+file:mix.exs&patternType=standard

Here is a search for public repositories that use Wallaby. You should be able to find a diversity of patterns and strategies for using Wallaby in those repositories.

I'll note tho that Wallaby is on v0.30, so some of the results probably show for much older versions (it shouldn't be much different, but will explain some discrepancies.)

Best of luck!

**Comment by mhanberg** [comment]
I have tweeted out a request for projects: https://twitter.com/elixir_wallaby/status/1567223686536593408

Hopefully others will chime in one some good ones!

**Reply by plpullen** [reply]
Thanks @mhanberg ! I so appreciate it!
